bin/oneoff/backfill_data/backfill_script_ids_in_teacher_feedbacks.rb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+# Backfill existing TeacherFeedbacks to set script_id based on script_level_id.
+
+require_relative '../../../dashboard/config/environment'
+
+def feedbacks_without_script_id
+  TeacherFeedback.where(script_id: nil)
+end
+
+def puts_count
+  puts "There are #{feedbacks_without_script_id.count} feedbacks without a script_id"
+end
+
+def update_script_ids
+  puts "starting backfilling script_ids"
+  feedbacks_without_script_id.find_in_batches(batch_size: 1000) do |batch|
+    print '.'
+    $stdout.flush
```

Comment:
I think we could set `$stdout.sync = true` once at the beginning of the script. [Example](https://github.com/code-dot-org/code-dot-org/blob/57c2cbd13e698146b4966a1f49f9aad1b92ca645/bin/i18n/i18n_script_utils.rb#L31).